client/react/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.3.0",
   "proxy": "http://localhost:4000",
   "scripts": {
-    "start": "react-scripts start"
+    "start": "react-scripts start",
+    "build": "react-scripts build"
```

Comment:
This is not strictly necessary right now but it won't hurt. I had been experimenting with a proper proxy locally.